src/autolaser.py

```diff
@@ -45,20 +75,24 @@ def get_max_contour(mask):
 # Finds both the center point of the balloon contour and also maps this to angles for the servos
 def find_center_point(max_contour):
     global L
+    global radius
     moments = cv2.moments(max_contour)
     center = (int(moments["m10"] / moments["m00"]), int(moments["m01"] / moments["m00"]))
 
     # Get the x and y coords of the center dot location on the image
     x_coord = int(moments["m10"] / moments["m00"])
     y_coord = 450 - int(moments["m01"] / moments["m00"])
 
+    # Find the distance
+    distance = distanceFinder(focalLength, KNOWN_RADIUS, radius)
```

Comment:
can you move this function call/distance variable to the code that draws the outline since that is where we will be using it?